src/lib/locales/de.ts

```diff
@@ -23,7 +23,7 @@ const de = {
 	'register.register': 'Registrieren',
 	'register.nametaken': 'Nutzername vergeben :(',
 	'register.nametaken.desc':
-		'Aber Du kannst noch immer den selben Anzeigenamen und das selbe Passwort nutzen',
+		'Aber Du kannst noch immer denselben Anzeigenamen und dasselbe Passwort nutzen',
```

Comment:
I've asked ChatGPT and it says that both "denselben" and "den selben" are correct. "den selben" is less formal, I want Dlool to be personal so please revert that one back to "den selben" and "das selbe"